### ISSUE TITLE
Configuration of default choices in Icingaweb2

### ISSUE THREAD

**Issue by linpopilan** [Icinga/icingaweb2#3477]
## Introduction
This is a feature request to this awesome monitoring community!

Is it possible to provide some kind of configuration page of default options in Icingaweb2?

An example: 

When doing an acknowledge, standard choice is: "Send Notification". In our NOC their typical use case is to disable that and enable "Persistent Comment" and "Sticky Acknowledgement". We would like a configuration option where you could change these kind of default values in Icinga.
![image](https://user-images.githubusercontent.com/7757435/41526445-8ec59afc-72e4-11e8-8320-091a1f075d8e.png)

## Solution

Provide a configuration page in the monitoring module where we could choose default values for every option in Icingaweb2, it would make it easy for all users to choose there default settings. Icingaweb2 could still be shipped with the current default options.

**Comment by cflinspach** [comment]
It's not a GUI configuration page but you can find the default options in in the monitoring module directory, in our cases it's  /usr/share/icingaweb2/modules/monitoring/application/forms/Command/Object/AcknowledgeProblemCommandForm.php

We disable "Send Notifications" as default as well by changing the `true` in line 133 (`'value'         => (bool) $config->get('settings', 'acknowledge_notify', true),` to false. The other options are in that file as well.

**Comment by marcelfischer** [comment]
Hey,

also see https://monitoring-portal.org/woltlab/index.php?thread/37141-change-sticky-acknowledgement-default/&postID=259374#post259374

**Comment by linpopilan** [comment]
@cflinspach yes, tho that is probably not the ideal way to change the source code "settings". When doing package upgrades, this will be an issue that either is overwritten or there is going to be a .save file next to AcknowledgeProblemCommandForm.php where the updated code is.

@marcelfischer I'll test this! That is a suitable workaround, but I still think that we should have a Configuration GUI for this kinds of settings. Preferably store the settings in a MySQL database, so that Icingaweb2 could be put under a load balancer etc.. 

Thanks both!

**Comment by lippserd** [comment]
Hi,

Thanks for the report. We consider this for one of our next feature releases.

Best,
Eric

**Comment by linpopilan** [comment]
@lippserd Thank you! Keep us posted regarding your consideration :)

As I mentioned above this would be great if the settings where to be stored in MySQL instead of a random file on disk.. Makes it's easier to loadbalance Icingaweb2 for future use cases.

**Comment by nilmerg** [comment]
This *might* be a candidate for v2.8. As already mentioned this is mostly of interest once we store such configuration in database. (#3770)

In the meantime, the available settings are now documented: https://icinga.com/docs/icingaweb2/latest/modules/monitoring/doc/03-Configuration/#default-settings
